dll/win32/iernonce/include/registry.h

```diff
@@ -13,6 +13,11 @@
 #include <atlcoll.h>
 #include <atlsimpcoll.h>
 
+#define FLAGS_NO_STAT_DIALOG 0x00000080
```

Comment:
"stat" ?

Reply:
yes, windows call this flag in the document "No Status Dialog"